package.json

```diff
@@ -415,11 +413,11 @@
         "permissionName": "ui-users.settings.transfertypes",
         "displayName": "Settings (Users): Can create, edit and remove transfer criteria",
         "subPermissions": [
-          "transfertypes.collection.get",
-          "transfertypes.item.delete",
-          "transfertypes.item.get",
-          "transfertypes.item.post",
-          "transfertypes.item.put",
+          "transfer-criterias.collection.get",
+          "transfer-criterias.item.delete",
+          "transfer-criterias.item.get",
+          "transfer-criterias.item.post",
+          "transfer-criterias.item.put",
```

Comment:
Is there a corresponding `okapiInterfaces` change here too? I don't know which interface these permissions are associated with, but it would surprise me if it's `login`, the only change indicated here.

Reply:
https://github.com/folio-org/mod-feesfines/commit/7905485237f1d647f3f0dcb7fbbc69fa44a3ccf1

At this point feesfines was version 15.0.  At later stage feesfines version was bumped to 15.1, 15.2.. (they should have gone straight to 16.0 as this truly a breaking change).

ui-users allows "16.1 17.0", so the change happened before the interfaces that ui-users currently allow.